README.md

```diff
@@ -59,19 +59,6 @@ npm run server:dev:hmr
 ```
 go to [http://0.0.0.0:3000](http://0.0.0.0:3000) or [http://localhost:3000](http://localhost:3000) in your browser
 
-# Nitrous Quickstart
```

Comment:
@gdi2290, I removed the Nitrous configuration completely because I think it's pretty opinionated and will therefore rather confuse new users than help them. I think that far more new users won't know what it is and will not use Nitrous. For those these are two unnecessary files which they will not know what to do with. What's your opinion?